scss/_variables.scss

```diff
@@ -797,29 +797,43 @@ $table-line-height:           1.25 !default; // Boosted mod
 
 $table-color:                 var(--#{$prefix}body-color) !default;
 $table-bg:                    $body-bg !default;
-// Boosted mod: no $table-accent-bg
+$table-accent-bg:             transparent !default;
 
 $table-th-font-weight:        null !default;
 
-$table-active-color:          $table-color !default;
-$table-active-bg-factor:      .1 !default;
-$table-active-bg:             rgba($black, $table-active-bg-factor) !default;
+$table-striped-color:                   $table-color !default;
+$table-striped-bg-factor:               .065 !default;
+$table-striped-bg:                      rgba($black, $table-striped-bg-factor) !default;
+$table-striped-hover-color:             $table-color !default; // Boosted mod
+$table-striped-hover-bg-factor:         .4 !default; // Boosted mod
+$table-striped-hover-bg:                rgba($black, $table-striped-hover-bg-factor) !default; // Boosted mod
+$table-variant-striped-bg-factor:       .2 !default; // Boosted mod
+$table-variant-striped-hover-bg-factor: .865 !default; // Boosted mod
 
-$table-hover-color:           $table-color !default;
-$table-hover-bg-factor:       .065 !default;
-$table-hover-bg:              rgba($black, $table-hover-bg-factor) !default;
+$table-active-color:                    $table-color !default;
+$table-active-bg-factor:                .135 !default; // Boosted mod
+$table-active-bg:                       rgba($black, $table-active-bg-factor) !default;
+$table-variant-active-bg-factor:        .6 !default; // Boosted mod
 
-$table-border-factor:         .4 !default;
-$table-border-width:          $border-width * .5 !default; // Boosted mod
-$table-border-color:          $gray-500 !default; // Boosted mod
+$table-hover-color:                     $table-color !default;
+$table-hover-bg-factor:                 .065 !default; // Boosted mod
+$table-hover-bg:                        rgba($black, $table-hover-bg-factor) !default;
+$table-variant-hover-bg-factor:         .2 !default; // Boosted mod
 
-// Boosted mod: no $table-striped-*
+$table-border-factor:                   .4 !default; // Boosted mod
+$table-border-width:                    $border-width * .5 !default; // Boosted mod
```

Comment:
For those 2 lines, there were modifications in Bootstrap since this PR has been created (I suppose). They now have:

```scss
$table-border-width:          var(--#{$prefix}border-width) !default;
$table-border-color:          var(--#{$prefix}border-color) !default;
```

Shouldn't we reuse custom properties as well?